### PR TITLE
Add ServiceMeshMemberRoll resource for Service Mesh usage.

### PR DESCRIPTION
**Pull request by ofirnash1**
##### Short description:
Add ServiceMeshMemberRoll resource for Service Mesh.

##### More details:
Add SMMR resource with ApGroup: maistra.io.

##### What this PR does / why we need it:
Service Mesh

### CODE REVIEW

File: ocp_resources/ServiceMeshMemberRoll.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```
Comment (rnetser) [inline ocp_resources/ServiceMeshMemberRoll.py:1]: Please rename the file to keep the same convention as other resources.

File: ocp_resources/resource.py [hunk 2]
```diff
@@ -279,6 +279,7 @@ class ApiGroup:
         LITMUS_IO = "litmuschaos.io"
         MACHINE_OPENSHIFT_IO = "machine.openshift.io"
         MACHINECONFIGURATION_OPENSHIFT_IO = "machineconfiguration.openshift.io"
+        MAISTRA_ISTIO_IO = "maistra.io"
```
Comment (rnetser) [inline ocp_resources/resource.py:282]: Either rename the variable or update the API group string; should match one another, so either:
MAISTRA_ISTIO_IO = "maistra.istio.io"
or
MAISTRA_IO = "maistra.io"

**Review by rnetser — CHANGES_REQUESTED** [review]

File: ocp_resources/resource.py (hunk 2 above)
Comment (ofirnash1) [reply ocp_resources/resource.py]: Changed to: MAISTRA_IO = "maistra.io"

File: ocp_resources/ServiceMeshMemberRoll.py (hunk 1 above)
Comment (ofirnash1) [reply ocp_resources/ServiceMeshMemberRoll.py]: Missed it, updated. Thanks:)

**Review by myakove — APPROVED** [review]

**Pull request merged**